src/gf/seismosizer.py

```diff
@@ -1692,20 +1740,38 @@ class RectangularSource(SourceWithDerivedMagnitude):
         optional=True,
         default=1)
 
+    gaussian_cutoff = Float.T(
+        optional=True,
+        help='if given, turn on 2D gaussian distributed slip on source '
```

Comment:
From this description I still dont get what the cut-off here does. I would need more explanation to know how to specify it.